app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SearchResultViewModel.java

```diff
@@ -22,13 +25,24 @@
 
     private static final int ELLIPSIZE_LIMIT_COUNT = 30;
 
+    @Retention(RetentionPolicy.SOURCE)
+    @IntDef({TYPE_TITLE, TYPE_DESCRIPTION, TYPE_SPEAKER})
+    public @interface Type {
+    }
+
+    public static final int TYPE_TITLE = 1;
+
+    public static final int TYPE_DESCRIPTION = 2;
+
+    public static final int TYPE_SPEAKER = 3;
```

Comment:
Thanks for refactoring!
I think it's better to replace `enum` instead of `int` because each type has own `iconResId`.
If we use `enum`, we can reduce `getIconResId()` method.

What do you think?